src/SignalR/clients/csharp/Http.Connections.Client/src/Internal/ITransport.cs

```diff
@@ -3,14 +3,15 @@
 
 using System;
 using System.IO.Pipelines;
+using System.Threading;
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.Connections;
 
 namespace Microsoft.AspNetCore.Http.Connections.Client.Internal
 {
     public interface ITransport : IDuplexPipe
     {
-        Task StartAsync(Uri url, TransferFormat transferFormat);
+        Task StartAsync(Uri url, TransferFormat transferFormat, CancellationToken cancellationToken = default);
```

Comment:
Are we ok with this breaking change? @anurse @davidfowl

Reply:
I was about to say this was an opportunity to include a default interface method that calls the old CancellationTokenless StartAsync overload if the new overload doesn't exist, but I think silently throwing away the CancellationToken is evil.

So I'm OK with this breaking change.

Reply:
This is pubternal so yeah, I'm OK with this.

Reply:
Yea it’s pubternal. It’s also probably time to make it internal

Reply:
https://github.com/aspnet/AspNetCore/issues/10169